modules/migration-compatibility-guidelines.adoc

```diff
@@ -20,29 +20,32 @@ modern operator:: The {mtc-short} Operator designed for modern platforms.
 control cluster:: The cluster that runs the {mtc-short} controller and GUI.
 remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
 
-
-[cols="1,2,2", options="header"]
 .{mtc-short} compatibility: Migrating from a legacy platform
 |===
-||{product-title} 4.5 or earlier |{product-title} 4.6 or later
-|Stable {mtc-short} version a|{mtc-short} {mtc-version}._z_
+| |{product-title} 3.11 |{product-title} 4.0 to 4.5 |{product-title} 4.6 or later
+
+|Stable {mtc-short} version
+|{mtc-short} v.1.7._z_
+|{mtc-short} v.1.7._z_
+|{mtc-short} v.1.7._z_
+
+| N/A
+| N/A
+|Legacy {mtc-short} v.1.7._z_ operator: Install manually with the `operator.yml` file.
 
-Legacy {mtc-version} operator: Install manually with the `operator.yml` file.
 [IMPORTANT]
```

Comment:
Unfrotunetly notes do not resolve well in tables withn regards to the real estate they take up. By removing the delimiter lines, the [IMPORTANT] syntax does not align with typical adminition formats. As the content in the table is short, how about:

> *Important:* This cluster cannot be the control cluster.

Reply:
[Example ](https://github.com/openshift/openshift-docs/blob/main/modules/installation-configuration-parameters.adoc#L2440C1-L2441C23) but with `Note:` in bold.